ext/curl/sync-constants.php

```diff
@@ -177,6 +177,62 @@ function getCurlConstants() : array
     return $constants;
 }
 
+/**
+ * Loads and parses the cURL version constants from the online documentation.
+ *
+ * The result is an associative array where the key is the constant name, and the value is a numeric array with:
+ * - the introduced version
+ * - the deprecated version (nullable)
+ * - the removed version (nullable)
+ *
+ * @return array
+ */
+function getCurlVersionConstants() : array
```

Comment:
Rather than duplicating this code, I think we should just adjust the `if (strpos($name, 'CURLOPT_') !== 0)` check in getCurlConstants()` to also return `CURL_VERSION_*`. Or is there some way in which these constants must be handled differently?

Reply:
Right! I was thinking to change these behavior in a separate PR, but I guess we can start here.

Reply:
It's done.